mobileapp/www/index.html

```diff
@@ -2,7 +2,7 @@
   <script>
       document.addEventListener('deviceready', onDeviceReady, false);
       // Warning: those locales are duplicated from locales-config.json
-      var locales = ['ar', 'cs', 'da', 'de', 'en', 'es', 'fr', 'hr', ,'id', 'it', 'ja', 'nl', 'pl', 'pt-br', 'ru', 'sk', 'sv', 'vi', 'zh-cn', 'zh-hk', 'zh-tw'];
+      var locales = ['ar', 'cs', 'da', 'de', 'en', 'es', 'et', 'fr', 'hr', ,'id', 'it', 'ja', 'nl', 'pl', 'pt-br', 'ru', 'sk', 'sv', 'vi', 'zh-cn', 'zh-hk', 'zh-tw'];
```

Comment:
Hmm, `el` (Greek) is missing, could you add it?

Reply:
Also what's that `'hr', ,'id',`?

Reply:
Fair enough!

Reply:
Nice catch with the double commas @veqtrus, makes me wonder if this thing is even used or if it's just been broken for a while?! 😂